fmriprep/workflows/bold/base.py

```diff
@@ -431,11 +431,17 @@ def init_func_preproc_wf(
         name='summary', mem_gb=DEFAULT_MEMORY_MIN_GB, run_without_submitting=True)
     summary.inputs.dummy_scans = dummy_scans
 
-    # CIfTI output: currently, we only support fsaverage{5,6}
-    cifti_spaces = set(s for s in output_spaces.keys() if s in ('fsaverage5', 'fsaverage6'))
-    fsaverage_den = output_spaces.get('fsaverage', {}).get('den')
-    if fsaverage_den:
-        cifti_spaces.add(FSAVERAGE_DENSITY[fsaverage_den])
+    # CIfTI output
+    cifti_spaces = set()
+    for space in output_spaces.keys():
+        if space == 'fsLR':
+            cifti_spaces = set(('fsLR',))
+            break
+        if space in ('fsaverage5', 'fsaverage6'):
+            cifti_spaces.add(space)
+        fsaverage_den = output_spaces.get('fsaverage', {}).get('den')
+        if fsaverage_den:
+            cifti_spaces.add(FSAVERAGE_DENSITY[fsaverage_den])
```

Comment:
@effigies, please correct me if I'm wrong but 'fsaverage' without a number is equivalent to `fsaverage7`, correct?
```suggestion
    fsaverage_den = output_spaces.get('fsaverage', {})
    if fsaverage_den:
        cifti_spaces.add(FSAVERAGE_DENSITY[fsaverage_den.get('den', '164k')])
```

also, this block should be outside the loop I think